msmbuilder/commands/fit.py

```diff
@@ -60,6 +60,12 @@ class MarkovStateModelCommand(FitCommand):
     _concrete = True
     _group = 'MSM'
 
+    def _ergodic_cutoff_type(self, erg):
```

Comment:
Silly question, but can someone point me to where I can figure out how this member function gets used?  I presume it's described in the numpydocs?